### PR TITLE
Add CODEOWNERS entries for genevb

### PR DESCRIPTION
**Pull request by genevb**

### CODE REVIEW

**Review by plexoos — APPROVED** [review]

**Comment by plexoos** [comment]
@nigmatkulov Grigory could you please unlock the merge button? After that you or Gene should be able to merge.

**Review by plexoos — APPROVED** [review]

**Review by nigmatkulov — APPROVED** [review]
Looks good

**Pull request merged**
